quantum/quantum.c

```diff
@@ -79,6 +79,10 @@ float default_layer_songs[][16][2] = DEFAULT_LAYER_SONGS;
 #    endif
 #endif
 
+#ifdef VIA_ENABLE
+#   include "via.h"
+#endif
+
```

Comment:
This block would be much better in `quantum.h`.  This would also remove the need to include `via.h` in every keyboard file that uses VIA.

Reply:
`via.h` does not need to be included in every keyboard that uses VIA, only the ones that extend base functionality (i.e. use EEPROM for their settings, like Satisfaction 75). I have made this change, but those keyboards still require explicit `#include "via.h"` so their EEPROM usage is unified for non-VIA builds.